blog/2025-01-21-Agama-11.mdx

```diff
@@ -0,0 +1,210 @@
+specifying the disk (or disks) that will be used as a base for the LVM. Agama will take care of
+creating all the needed partitions, honoring any other aspect of the configuration in the process.
+Find a more detailed explanation with examples at the [corresponding
+section](https://agama-project.github.io/docs/user/unattended/storage#generating-physical-volumes)
+of the Agama documentation.
+
+On the other hand, now it is possible to specify TPM-based unlocking of the encrypted devices as
+part of the Agama storage configuration. Thus, users of unattended installation can also deploy
+fully encrypted systems based on TPMv2.
+
+## Automatic generation of documentation and shell completion {#generate}
+
+A comprehensive and up-to-date documentation is key for a project like Agama, especially for users
+of the command-line interface and the HTTP API. And the best way to ensure the documentation is
+always in sync with the current version of Agama is to generate it automagically from the source
+code.
+
+In the case of the CLI, the manual pages, its Markdown variant and also the files needed for
+[shell completion](https://en.wikipedia.org/wiki/Command-line_completion) are all generated from
+sources. You can see the always current result of the Markdown version at the [corresponding
+section](https://agama-project.github.io/docs/user/cli) of the Agama web page.
+
+The HTTP API is also automatically documented via an [OpenAPI](https://www.openapis.org/)
+specification. This will help anyone interested in integrating Agama into any solution or
+infrastructure or even in creating its own client application for Agama.
+
+## More changes under the hood {#techdetails}
+
+As you can imagine, the above list of features is far for representing everything that has changed
+from Agama 10. As usual, the new version also includes many bug fixes and small improvements. And
+we took the opportunity to update to the latest version of the three programming languages used in
+Agama, including all used libraries.
+
+We also gave some love to the Agama Live ISO. On the one hand, we revisited the list of included
+drivers, resulting in a smaller image that actually supports more hardware setups. On the other hand
+we tried to switch the graphical stack from X11 to Wayland. Although we didn't succeed due to
+technical problems related to Firefox's kiosk mode, we will keep trying (of course, any help is
+welcome).
+
+Other Agama change that may not be obvious to all users is the introduction of some changes to ease
+the creation of automated integration test. That helps the openSUSE and SUSE QA teams in their
+invaluable effort to ensure a smoother experience to all users.
+
+## Just another step {#conclusion}
+
+We are already working on the next version of Agama and your feedback may be useful to decide in
+which aspects we should focus. So do not hesitate to give Agama a try using our [Live ISO testing
```

Comment:
"try using our **latest** images"

Just to make clear that the link points to the latest build, not to the Agama 10 release snapshot. In the future it will contain Agama 12...